### ISSUE TITLE
Compile error

### ISSUE THREAD

**Issue by nieldk** [gabriel-tenma-white/mculib#2]
This library is included in NanoVNA2 firmware

But, it errors when compiling

```
Line 28 
		static constexpr int cpCurrent = 15;

Needs to be
		int cpCurrent = 15;
```

And I can compile successfully

**Comment by douardda** [comment]
It's unclear to me why most ADF4350Driver "config-like" attributes are declared as "static constexpr"m especially the constexpr qualifier. Why do these values required at compile time? And why are these static members? Why should be shared values between multiple instances of the ADF4350Driver class?

**Comment by douardda** [comment]
Also why the presence of the unused `bool was_pwdn;' variable in sendConfig()?

**Comment by douardda** [comment]
I believe this can now be closed, see PR #1

**Comment by wutje** [comment]
The reason to make them compile time constant was to save a few bits of flash, RAM and CPU cycles.
For a new developer it is also easier to see they are not ever changed.

On the other hand this is a library and perhaps flexibility is more important here.

Having "was_pwdn" is an error on my side.
